assets/schema_input.json

```
@@ -44,14 +46,16 @@
                         "format": "file-path",
                         "pattern": "^\\S+\\.(fasta|fas|fa|faa)(\\.gz)?$"
                     },
+                    {
+                        "exists": true
+                    },
```

```suggestion
                        "format": "file-path",
                        "pattern": "^\\S+\\.(fasta|fas|fa|faa)(\\.gz)?$",
                        "exists": true
                    },
```

This is better because you don't want to check if a file exists if it isn't a file :)